.gitmodules

```diff
@@ -10,3 +7,7 @@
 [submodule "modules/afamqp/rabbitmq-c"]
 	path = modules/afamqp/rabbitmq-c
 	url = https://github.com/alanxz/rabbitmq-c.git
+[submodule "modules/afmongodb/mongo-c-driver"]
```

Comment:
I'm not sure. Fortunately for us, they have just released the 1.2 stable few days ago. Unfortunately, I found and fixed a UNIX domain socket bug in mongo-c-driver and I'm not sure when they will merge my pull request. I don't think this is an import issue, we could simply disregard that for now and use something stable.